src/config.rs

```diff
@@ -166,6 +171,26 @@ impl OrchestratorConfig {
             warn!("no chunker configs provided");
         }
 
+        if config.allowed_headers_passthrough.is_none() {
+            warn!("No allowed headers specified");
+        }
+
+        // Add default headers to allowed_headers list
+        debug!("Adding default headers");
+        // config.allowed_headers_passthrough.unwrap_or_default().extend(DEFAULT_ALLOWED_HEADERS.iter().map(|h| h.to_string()));
+        if let Some(passthroughs) = &mut config.allowed_headers_passthrough {
+            passthroughs.extend(DEFAULT_ALLOWED_HEADERS.iter().map(|h| h.to_string()));
+            // De-duplicate
+            passthroughs.dedup();
+        } else {
+            config.allowed_headers_passthrough = Some(
+                DEFAULT_ALLOWED_HEADERS
+                    .iter()
+                    .map(|h| h.to_string())
+                    .collect(),
+            );
+        }
```

Comment:
Update and simplify per other changes.

```suggestion
        if config.passthrough_headers.is_empty() {
            warn!("no passthrough headers specified");
        }

        // Add default headers to allowed_headers list
        debug!("Adding default headers");
        config
            .passthrough_headers
            .extend(DEFAULT_PASSTHROUGH_HEADERS.iter().map(|v| v.to_string()));
```